CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Next release
 
+- dev: fix rpc test failing
```

Comment:
did you check the other crates in the repo? In order to be sure the same problem is not present somewhere else?

Reply:
Adding one more which i found while testing all crates.
`mp-felt/std`